plugins/eni/commands/commands.go

```diff
@@ -97,6 +111,15 @@ func add(args *skel.CmdArgs, engine engine.Engine) error {
 			return err
 		}
 		log.Debugf("IPV6 Gateway IP: %v", ipv6Gateway)
+		_, ipv6net, err := net.ParseCIDR(ipv6Address)
+		if err != nil {
+			return errors.Wrapf(err, "parsing ipv6 failed:%s", ipv6Address)
```

Comment:
Same here, please make provide more context here: 

`"add eni: failed to parse ipv6 gateway: %s"`